components/ide-service/pkg/server/server.go

```diff
@@ -490,9 +490,9 @@ func (s *IDEServiceServer) ResolveWorkspaceConfig(ctx context.Context, req *api.
 		resp.IdeSettings = ideSettingsEncoded.String()
 	}
 
+	// TODO figure out how to make it configurable on IDE level, not hardcoded here
```

Comment:
we deal with it later, it will need some kind of metadata configuration in the image itself that we can compare it against .gitpod.yml and configs here and decide